config/stack/manifests/resources/accountclass.resource.yaml

```diff
@@ -8,4 +8,4 @@ overview: |
 readme: |
  Cloud-specific resource classes are used to define a reusable configuration for a specific managed service. This class provides a storage account, which is satisfied by [Azure Blob Storage Accounts](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview).
 
- Full documentation can be found in the Crossplane [API Reference docs](https://doc.crds.dev/github.com/crossplane/provider-azure/storage.azure.crossplane.io_accountclasses.yaml).
+ Full documentation can be found in the Crossplane [API Reference docs](https://doc.crds.dev/github.com/crossplane/provider-azure/config/crd/storage.azure.crossplane.io_accountclasses.yaml@v0.8.0).
```

Comment:
we should update our release-process to include this new type of transformation, but looks good!
https://doc.crds.dev/github.com/crossplane/provider-azure/config/crd/storage.azure.crossplane.io_accountclasses.yaml@v0.8.0

Reply:
For sure :) Also, it may be desirable to make the url resolution default to latest semver rather than latest master, which I would be happy to explore!